book/src/node-driver-registrar.md

```diff
@@ -0,0 +1,26 @@
+# CSI `node-driver-registrar`
+
+## Status and Releases
+
+**Git Repository:** https://github.com/kubernetes-csi/node-driver-registrar
+
+**Status:** GA/Stable
+
+Latests stable release | Branch | Documentation | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
+--|--|--|--|--|--|--
+[node-driver-registrar v1.0.2](https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v1.0.2) | [release-1.0](https://github.com/kubernetes-csi/node-driver-registrar/tree/release-1.0) | [Usage Docs](https://github.com/kubernetes-csi/node-driver-registrar/blob/release-1.0/README.md) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-node-driver-registrar:v1.0.2 | v1.13 | -
+[driver-registrar v0.4.2](https://github.com/kubernetes-csi/driver-registrar/releases/tag/v0.4.2) | [release-0.4](https://github.com/kubernetes-csi/driver-registrar/tree/release-0.4) | - | [v0.3.0](https://github.com/container-storage-interface/spec/releases/tag/v0.3.0) | quay.io/k8scsi/driver-registrar:v0.4.2 | v1.10 | -
+
+## Description
+
+The CSI `node-driver-registrar` is a sidecar container that fetches driver information (using `NodeGetInfo`) from a CSI endpoint and registers it with the kubelet on that node using the [kubelet plugin registration mechanism](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-registration).
+
+## Usage
+
+Kubelet directly issues CSI `NodeGetInfo`, `NodeStageVolume`, and `NodePublishVolume` calls against CSI drivers. It uses the [kubelet plugin registration mechanism](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#device-plugin-registration) to discover the unix domain socket to talk to the CSI driver. Therefore, all CSI drivers should use this sidecar container to register themselves with kubelet.
+
+For detailed information (binary parameters, etc.), follow the `Usage Docs` link in the table above for the appropriate version.
```

Comment:
Maybe we should not have a usage docs column? The other sidecars don't have it and just point to the readme on head

Reply:
Isn't the plan to add usage docs to all the repos?

Reply:
I guess it's a little redundant that the usage docs is just the README, which is the same link as the branch

Reply:
I can buy that will remove.